build fail with macos

<!-- Describe your issue and tell us how to reproduce it (include any useful information). -->
OS name: "mac os x", version: "10.14.4"

```
autoreconf: configure.ac: not using Gettext
tools/mockup.sh: line 11: syntax error near unexpected token `;;'
tools/mockup.sh: line 11: `		;;'
make: *** [tools/test/gen_test.c] Error 2
```

https://travis-ci.org/arowser/ln_nightly/jobs/601746053#L1443

--- a/tools/check-spelling.sh
+++ b/tools/check-spelling.sh
@@ -6,7 +6,7 @@ if git --no-pager grep -nHiE 'l[ightn]{6}g|l[ightn]{8}g|ilghtning|lgihtning|lihg
     exit 1
 fi
 
-if git --no-pager grep -nHiE 'ctlv' | grep -v 'enctlv' -- . ':!tools/check-spelling.sh'; then
+if git --no-pager grep -nHiEP '(?<!en)ctlv' -- . ':!tools/check-spelling.sh'; then
     echo "It's check lock time verify, not check time lock verify!" >&2
     exit 1
 fi

--- a/tools/mockup.sh
+++ b/tools/mockup.sh
@@ -8,10 +8,8 @@ fi
 UPDIRNAME=$(dirname "$(dirname "$1")")
 shift
 
-if [ $# -eq 0 ]; then
-    # With no args, read stdin to scrape compiler output.
-    # shellcheck disable=SC2046
-    set -- $(while read -r LINE; do
+function process_line {
+	LINE=$1
 	case "$LINE" in
 	    *undefined\ reference\ to*)
 		# file.cc:(.text+0x10): undefined reference to `foo()'
@@ -29,9 +27,16 @@ if [ $# -eq 0 ]; then
 		echo "${LINE%\"*}"
 		;;
 	    *)
-		continue
+		return
 		;;
-	esac; done | LC_ALL=C sort -u)
+	esac
+}
+
+if [ $# -eq 0 ]; then
+    # With no args, read stdin to scrape compiler output.
+    # shellcheck disable=SC2046
+    set -- $(while read -r LINE; do
+	process_line "$LINE"; done | LC_ALL=C sort -u)
 fi
 
 for SYMBOL; do

